features/bonding/examples/mco-bonding.md

```diff
@@ -0,0 +1,72 @@
+# Bonding using machine-config-operator (MCO)
+
+Project page [machine-config-operator](https://github.com/openshift/machine-config-operator)
+
+This document focusses on using `Machine Config operator (MCO)` for interface bonding and vlan configurations on a node. `MachineConfig` CRD is used to write configuration files onto the node. It expects the file contents to be supplied as encoded data per current ignition specifications, the encoding is done via `base64` in this document.
+
+### Sample bond interface file
```

Comment:
Can you please use the script we have in this folder to create the ignition config and not do it manually?

Reply:
The intention here is to include a command based alternative method for generating ignition configs, since the documented script approach requires specific versions and modules to be installed as prerequisites. Make sense?
I've added a pointer link directing to script document in order to highlight both methods.